src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java

```diff
@@ -618,7 +632,7 @@ public void testCreateLoadInNodeBreaker() throws Exception {
         loadCreationInfos.setBusOrBusbarSectionId("notFoundBusbarSection");
         loadCreationInfosJson = objectWriter.writeValueAsString(loadCreationInfos);
         mockMvc.perform(post(uriString, TEST_NETWORK_ID).content(loadCreationInfosJson).contentType(MediaType.APPLICATION_JSON))
-            .andExpectAll(status().is5xxServerError(), content().string(new NetworkModificationException(CREATE_LOAD_ERROR, "Busbar section notFoundBusbarSection not found.").getMessage())).andReturn();
+            .andExpectAll(status().is5xxServerError(), content().string(new NetworkModificationException(CREATE_LOAD_ERROR, "java.lang.NullPointerException").getMessage())).andReturn();
```

Comment:
Add a check in your getPosition method to test if network.getBusbarSection return null, and throw an explicit exception with a message like before ...